### PR TITLE
Fix(form-validations): Attach validation text to different parent on …

### PR DESCRIPTION
**Pull request by literat**
…checkbox

refs #DS-787

### CODE REVIEW

**Comment by literat** [comment]
Screenshot 2023-05-22 at 18 23 04
Screenshot 2023-05-22 at 18 23 24

@adamkudrna @crishpeen I need to add `display: block` to this `span` element or use a different class name (the class is general but the block is specific only for the checkbox). Any suggestion?

**Comment by literat** [comment]
> Screenshot 2023-05-22 at 18 23 04 Screenshot 2023-05-22 at 18 23 24
> @adamkudrna @crishpeen I need to add `display: block` to this `span` element or use a different class name (the class is general but the block is specific only for the checkbox). Any suggestion?

Added missing `data-element` selector to CheckboxField.

File: packages/web/src/scss/components/CheckboxField/_CheckboxField.scss [hunk 1]
```diff
@@ -51,7 +51,8 @@ $_field-name: 'CheckboxField';
     }
 }
 
-.CheckboxField__message {
+.CheckboxField__message,
+.CheckboxField [data-element='validator_message'] {
```
Comment (crishpeen) [inline packages/web/src/scss/components/CheckboxField/_CheckboxField.scss:55]:
```suggestion
.CheckboxField > .CheckboxField__text > [data-element='validator_message'] {
```

**Review by crishpeen — DISMISSED** [review]
Unfortunately, it doesn't work. When I check the checkbox in the demo, I get an error message.

File: packages/web/src/scss/components/CheckboxField/_CheckboxField.scss (hunk 1 above)
Comment (literat) [reply packages/web/src/scss/components/CheckboxField/_CheckboxField.scss]: Sorry 🤦

**Pull request merged**